alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -78,11 +93,13 @@ class AlertEngineImplTest extends FunSuite with BaseSolrCloudTest with MockitoSu
       new AlertEngineImpl(
         solrClient,
         new NotificationServiceFactory(mailNotificationService, slackNotificationService))
-    val result = engine.triggeredAlert(TEST_COLLECTION, alert).get
+    val result: TriggeredAlert = engine.triggeredAlert(TEST_COLLECTION, alert).get
     assertResult(alert)(result.rule)
     // @TODO why is this 2 instead of 1 sometimes?
     assert(result.documents.lengthCompare(0) > 0)
     assert(result.applicationName == TEST_COLLECTION)
+    //print out to see the result
+    println(NotificationFormatter.formatMessage(result))
```

Comment:
No need to print this out.

Reply:
The formatter should be tested somewhere else.

Reply:
Will remove this as well, will also update the formatter.